core/tests/helper.py

```diff
@@ -0,0 +1,14 @@
+# !/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""Helper methods for testing purpose"""
+
+
+def build_template_model_dict_data(sample_index=0):
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line](https://app.codacy.com/manual/lucas625/Microservice/pullRequest?prid=5220967)